startup-scripts/k8s-init.sh

```diff
@@ -43,13 +43,14 @@ cd /app && superset db upgrade
 echo_step "1" "Complete" "Applying DB migrations"
 
 # Create an admin user
-echo_step "2" "Starting" "Setting up admin user ( admin / $ADMIN_PASSWORD )"
+echo_step "2" "Starting" "Setting up admin user"
 superset fab create-admin \
               --username admin \
               --firstname Superset \
               --lastname Admin \
               --email admin@superset.com \
               --password "$ADMIN_PASSWORD"
+superset fab reset-password --username admin --password "$ADMIN_PASSWORD"
```

Comment:
question (non-blocking): Where does the password variable come from?

Reply:
It's set via the charm config. Or in the case of the Rock (without charm) you can set it as a variable for the service.